test/includes/lxc.sh

```diff
@@ -0,0 +1,52 @@
+# lxc CLI related test helpers.
```

Comment:
This comment is against commit https://github.com/lxc/lxd/pull/3673/commits/5c4c80e0fdc1cab20a2673f996092322b4d738d0

Which says "tests: split out network-related helper functions" but then goes on to include functions such as:
 - lxc
 - lxc_remote
 - spawn_lxd
 - respawn_lxd
 - kill_lxd
 - shutdown_lxd
 - cleanup
 - wipe

Which have nothing to do with networking.